src/config.rs

```diff
@@ -3,16 +3,14 @@
 //! This module provides the configuration structure and related types for
 //! controlling the behavior of the HTTP Gateway.
 
+use anyhow::anyhow;
```

Comment:
Please can we keep `anyhow` out of the library code. It's great when there are no consumers but trying to include anyhow errors into any other error handling mechanism than itself is a right pain.

Reply:
https://github.com/holochain/hc-http-gw/pull/25/commits/e9c81a55b96d531775aab730e954f988136b34dc